spec/features/tournaments_spec.rb

```diff
@@ -0,0 +1,24 @@
+require "rails_helper"
+
+RSpec.describe 'tournaments', :type => :feature do
+  let(:password) { 'asdfasdf' }
+  let(:admin) { create :admin, :password => password }
+  let!(:tournament1) { create :tournament }
+  let!(:tournament2) { create :tournament }
+
+  before(:each) do
+    visit new_user_session_path(year: admin.year)
+    fill_in 'Email', with: admin.email
+    fill_in 'Password', with: password
+    click_button 'Sign in'
+  end
+
+  it 'saves orders' do
+    visit tournaments_path(year: admin.year)
+    expect(page).to have_text 'Order'
+    fill_in "ordinals[#{tournament1.id}]", with: 2
+    fill_in "ordinals[#{tournament2.id}]", with: 1
+    click_button 'Update Order'
+    expect(page).to have_text 'Order updated'
```

Comment:
Thanks a ton for including a test with this change! Could you have this actual check the tournaments' data to see if they were properly updated? If we only check the success message, it leaves open the possibility that the saving works, but the logic might not actually successfully update the tournaments themselves.

Reply:
Thanks for the comment! I originally had that check, but the I think the test checks before the db gets updated. In my previous work, we used "wait_for_ajax" in rspec feature tests for these cases, but somehow it wasn't working on the congress site... Maybe another idea is to have a separate unit test. What do you think?

Reply:
There's definitely a lot I don't know about rspec and testing in Ruby on Rails, but I dug around a little bit and figured out how to check. What I learned is that if you just check, say, `tournament1.ordinal`, you don't get an update, because clicking the update order button updates the items in the test db, not the items _in memory_. But there's a method called `reload` that gets you the updated value.

(in `tournaments_spec.rb`)
```ruby
# Use "reload" to check the updated attributes of objects
# @see https://stackoverflow.com/a/38164881/399077
expect(tournament1.reload.ordinal).to be 2
expect(tournament2.reload.ordinal).to be 1
```

Try this out—it worked for me, and gives a little more assurances that the tournaments are actually getting the ordinal value we want.

Reply:
Reload worked! Sorry I saw your code examples after I pushed my updates. I read your comment on my phone, and I must have missed the codes in the end. Should I make them shorter like you did?

Reply:
Nah, that's okay. I do think the inline reload has a nice flow to it, but that's a small matter. Thank you for updating the test -- I think this gives us a very nice assurance now about being able to save values correctly.